### PR TITLE
SaveCourse can now support multiple courses

### PR DESCRIPTION
**Pull request by the-daniel-rothig**
### Context

Publish courses when org enrichment is published 

### Changes proposed in this pull request

- change save course to a HTTP PUT to api/courses
- increase request size limit
- accept a list of courses rather than a single course

### CODE REVIEW

**Comment by fofr** [comment]
I assume this doesn't publish every course draft?

File: src/api/Controllers/CoursesController.cs [hunk 1]
```diff
@@ -33,36 +33,36 @@ public CoursesController(ICourseDbContext courseDbContext, ILogger<CoursesContro
             _logger = logger;
         }
 
-        [HttpPost("{providerCode}/{courseCode}")]
+        [HttpPut("")]
         [ApiTokenAuth]
-        public async Task<IActionResult> SaveCourse(string providerCode, string courseCode, [FromBody]Course course)
+        [RequestSizeLimit(100_000_000_000)]
+        public async Task<IActionResult> SaveCourses([FromBody]IList<Course> courses)
```
Comment (defong) [inline src/api/Controllers/CoursesController.cs]: You going to add the counterpart of this in the client?

Should there be additional validation to ensure that the list of courses are all for one provider?

Comment (the-daniel-rothig) [reply src/api/Controllers/CoursesController.cs]: Hmm -- not sure S+C needs to enforce this as a restriction. It just happens to be how MC uses it

**Comment by defong** [comment]
Merging this in as it stole version number on nuget

**Pull request merged**